### PR TITLE
fix the readme

### PR DESCRIPTION
**Pull request by Polynomia**

### CODE REVIEW

File: apps/tfnet/image_classification_inference.ipynb [hunk 1]
```diff
@@ -97,6 +97,7 @@
      "text": [
       "Adding /home/yang/sources/zoo/zoo/target/analytics-zoo-0.2.0-SNAPSHOT-jar-with-dependencies.jar to BIGDL_JARS\n",
       "Adding /home/yang/sources/zoo/zoo/target/analytics-zoo-0.2.0-SNAPSHOT-jar-with-dependencies.jar to SPARK_CLASSPATH\n",
+      "If there is an error when init spark context because SPARK_CLASSPATH is deprecated in Spark 1.0+, you can unset SPARK_CLASSPATH\n",
```
Comment (yangw1234) [inline apps/tfnet/image_classification_inference.ipynb]: Is this line generated by the program or added by you?

Comment (hkvision) [reply apps/tfnet/image_classification_inference.ipynb]: Hahaha should be added by her. The program won't be that smart to generate such remark :)

Comment (hkvision) [reply apps/tfnet/image_classification_inference.ipynb]: Synced offline. No need this extra comment...

File: apps/tfnet/image_classification_inference.ipynb [hunk 2]
```diff
@@ -97,6 +97,7 @@
      "text": [
       "Adding /home/yang/sources/zoo/zoo/target/analytics-zoo-0.2.0-SNAPSHOT-jar-with-dependencies.jar to BIGDL_JARS\n",
```
Comment (hkvision) [inline apps/tfnet/image_classification_inference.ipynb]: Remove these two lines

**Pull request merged**